webpack/assets/javascripts/jquery.ui.custom_spinners.js

```diff
@@ -0,0 +1,115 @@
+const megabyte = 1024 * 1024;
+const gigabyte = 1024 * megabyte;
+
+$(function () {
+  $.widget('ui.limited_spinner', $.ui.spinner, {
+    options: {
+      softMaximum: 0,
+      errorTarget: null
+    },
+    _validate: function () {
```

Comment:
This doesn't seem to trigger on manual changes of the field (ie not using the spinner)